src/car/car.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { CreateCarDto } from './dto/create-car.dto';
+import { UpdateCarDto } from './dto/update-car.dto';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { Car } from '@prisma/client';
+
+@Injectable()
+export class CarService {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  async create(createCarDto: CreateCarDto): Promise<Car> {
+    const { marca, modelo, year, km, color, ac, pasajeros, cambios } =
+      createCarDto;
+    const newCar = await this.prismaService.car.create({
+      data: {
+        marca,
+        modelo,
+        year,
+        km,
+        color,
+        ac,
+        pasajeros,
+        cambios,
+      },
+    });
```

Comment:
I think you could pass the createCarDto to the `data` property in the same way you do in the `update` method

```suggestion
    const newCar = await this.prismaService.car.create({
      data: createCarDto,
    });
```